src/index.ts

```diff
@@ -1,214 +1,127 @@
+    authority: {
+      /**
+       * Is my program allowed to do what I want to do?
+       */
+      isGranted: (query?: Query) => Promise<
+        { granted: true } | { granted: false; reason: string }
+      >
+
+      provide: () => Promise<void>
+      request: (options: RequestOptions) => Promise<void>
+    }
 
     /**
-     * Try to create a `Session` based on capabilities.
+     * Components used to build this program.
      */
-    session: (username: string) => Promise<Maybe<Session>>
-  }
-
-  /**
-   * Configuration used to build this program.
-   */
-  configuration: Configuration
-
-  /**
-   * Components used to build this program.
-   */
-  components: Components
-
-  /**
-   * Various file system methods.
-   */
-  fileSystem: FileSystemShortHands
+    components: Components<Annex>
 
-  /**
-   * Existing session, if there is one.
-   */
-  session: Maybe<Session>
-}
+    /**
+     * Configuration used to build this program.
+     */
+    configuration: Configuration
 
+    /**
+     * Various file system methods.
+     */
+    fileSystem: FileSystemShortHands
+  }
+  & Events.ListenTo<Events.All>
 
 export enum ProgramError {
   InsecureContext = "INSECURE_CONTEXT",
-  UnsupportedBrowser = "UNSUPPORTED_BROWSER"
+  UnsupportedBrowser = "UNSUPPORTED_BROWSER",
 }
 
-
-export type ShortHands = {
-  accountDID: (username: string) => Promise<string>
-  agentDID: () => Promise<string>
-  sharingDID: () => Promise<string>
-}
-
-
 export type FileSystemShortHands = {
   addSampleData: (fs: FileSystem) => Promise<void>
 
   /**
-   * Load the file system of a given username.
+   * Load the file system associated with the account system.
    */
-  load: (username: string) => Promise<FileSystem>
-
-  /**
-   * Recover a file system.
-   */
-  recover: (params: RecoverFileSystemParams) => Promise<{ success: boolean }>
+  load: (opts?: { local?: boolean }) => Promise<FileSystem>
 }
 
-
-
-// ENTRY POINTS
-
+//////////////////
+// ENTRY POINTS //
+//////////////////
 
 /**
  * 🚀 Build an ODD program.
  *
- * This will give you a `Program` object which has the following properties:
- * - `session`, a `Session` object if a session was created before.
- * - `auth`, a means to control the various auth strategies you configured. Use this to create sessions. Read more about auth components in the toplevel `auth` object documention.
- * - `capabilities`, a means to control capabilities. Use this to collect & request capabilities, and to create a session based on them. Read more about capabilities in the toplevel `capabilities` object documentation.
- * - `components`, your full set of `Components`.
+ * This will give you a `Program` object which will your main interaction point.
+ *
+ * This gives you three systems to work with:
+ * - `account`, the account system, use this to register an account.
+ * - `authority`, the authority system, request or provide authority to parts of the (or entire) file system and account system.
```

Comment:
```suggestion
 * - `authority`, the authority system, request or provide authority to the account system and parts of (or the entire) file system.
```

Rearranged this a bit. Question though, can authority be requested for parts of the account system? If so, would recommend this instead:

```
* - `authority`, the authority system, request or provide authority to parts of (or the entire) file system and account system.
```

Reply:
Yeah, parts of both the file and account system.

Reply:
Gotcha, I updated the suggestion to reflect that. 👌